<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IseCaliSettingWidget</class>
 <widget class="QWidget" name="IseCaliSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">IseCaliSettingWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>5</y>
     <width>560</width>
     <height>287</height>
    </rect>
   </property>
   <property name="title">
    <string>自动校准设置</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
   <widget class="QCheckBox" name="consumables_change_ckbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>73</y>
      <width>511</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>耗材更换</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1457</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1</width>
     <height>792</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1549</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>792</y>
     <width>1554</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>1086</x>
     <y>5</y>
     <width>458</width>
     <height>783</height>
    </rect>
   </property>
   <property name="title">
    <string>校准检查范围</string>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>73</y>
      <width>421</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>斜率变化允许值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="compensated_edit">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>40</height>
        </size>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>512</width>
     <height>287</height>
    </rect>
   </property>
   <property name="title">
    <string>校准结果管理</string>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>73</y>
      <width>461</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>校准周期</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLineEdit" name="time_edit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="maxLength">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>天</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>254</y>
     <width>512</width>
     <height>534</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
   <widget class="QCheckBox" name="cali_auto_mask_ckbox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>73</y>
      <width>461</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>校准屏蔽</string>
    </property>
    <property name="checks" stdset="0">
     <string notr="true">push</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>254</y>
     <width>560</width>
     <height>285</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
   <widget class="QCheckBox" name="timeout_cali_type_ckbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>73</y>
      <width>501</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>校准结果过期</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>502</y>
     <width>560</width>
     <height>286</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>501</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>质控失败</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="titles" stdset="0">
     <string notr="true">sub_tit</string>
    </property>
    <property name="weights" stdset="0">
     <string notr="true">bold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="union_qc_cali_mode_ckbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>169</y>
      <width>511</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>联合质控品失控</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="single_qc_cali_mode_ckbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>109</y>
      <width>511</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>单质控品失控</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
